Add sentence on graphing partial data (#27594)
DataDog/documentation: content/en/dashboards/widgets/timeseries.md

## Display options

> Graphs can be displayed as lines, areas, and bars. Datadog graphs aggregate data over [intervals][15] such as sum every hour. By default, the last incomplete time interval in a timeseries graph is shaded and labeled as "partial data"."

Reviewer: ```suggestion
Graphs can be displayed as lines, areas, and bars. Datadog graphs aggregate data over [intervals][15] such as sum every hour. By default, the last incomplete time interval in a timeseries graph is shaded and labeled as "partial data".
```

> Graphs can be displayed as lines, areas, and bars. Datadog graphs aggregate data over intervals such as sum every hour. By default, the last time interval in a timeseries graph with incomplete data is shaded and labeled it as ‘partial data’.

Author: ```suggestion
Graphs can be displayed as lines, areas, and bars. Datadog graphs aggregate data over intervals such as sum every hour. By default, the last time interval in a timeseries graph with incomplete data is shaded and labeled it as "partial data".
```

> Graphs can be displayed as lines, areas, and bars. Datadog graphs aggregate data over intervals such as sum every hour. By default, the last time interval in a timeseries graph with incomplete data is shaded and labeled it as "partial data".

Reviewer 2: I think the last sentence here could be adjusted to sound more accurate to how we calculate (nit: the "it" seems like a typo) -

```suggestion
Graphs can be displayed as lines, areas, and bars. Datadog graphs aggregate data over [intervals](https://docs.datadoghq.com/dashboards/functions/rollup/) such as sum every hour. By default, the last incomplete time interval in a timeseries graph is shaded and labeled as "partial data"."
```
